net-misc/httpie/httpie-1.0.0.ebuild

```diff
@@ -5,16 +5,23 @@ EAPI=7
 
 PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
 
-inherit distutils-r1
+inherit bash-completion-r1 distutils-r1
 
 DESCRIPTION="Modern command line HTTP client"
 HOMEPAGE="https://httpie.org/ https://pypi.org/project/httpie/"
 SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-DEPEND=""
-RDEPEND=">=dev-python/pygments-2.1.3[${PYTHON_USEDEP}]
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="dev-python/pygments[${PYTHON_USEDEP}]
 	>=dev-python/requests-2.11.0[${PYTHON_USEDEP}]"
+
+python_install_all() {
+	newbashcomp ${FILESDIR}/${PN}-completion.bash ${PN}
+	bashcomp_alias ${PN} http
```

Comment:
I think you don't need an alias if you install completion file as `http` instead of `${PN}`. in fact, package only installs binary as `http` so it should not use `${PN}` here, as there is no `httpie` binary in `/usr/bin`

fish completions are installed like regular files into `/usr/share/fish/vendor_completions.d` and should be named `binaryname.fish`
this example should do the right thing:
```sh
insinto /usr/share/fish/vendor_completions.d
newins extras/httpie-completion.fish http.fish
```